Python/PRP/Region/pyVolumeIsect.h

```diff
@@ -18,8 +18,10 @@
 #define _PYVOLUMEISECT_H
 
 #include "PyPlasma.h"
+#include <PRP/Region/plVolumeIsect.h>
 
 PY_WRAP_PLASMA(VolumeIsect, class plVolumeIsect);
+PY_WRAP_PLASMA(SinglePlane, class plConvexIsect::SinglePlane);
```

Comment:
This probably makes more sense as a `PY_WRAP_PLASMA_VALUE`, using copying semantics rather than pointers and ownership...